getting_started/access_your_services.adoc

```diff
@@ -113,6 +113,74 @@ Alternatively, you can use a custom domain:
 $ oc expose service <service-name> -l route=external --name=<custom-route-name> --hostname=<custom-domain>
 ----
 
+[[expose-tcp-services]]
+== Expose TCP Services
+
+{product-title} xref:../architecture/networking/routes.adoc[routes] expose applications by proxying traffic through HTTP/HTTPS(SNI)/TLS(SNI)
+to xref:../architecture/core_concepts/pods_and_services.adoc[pods and services]. A link:https://kubernetes.io/docs/concepts/services-networking/#loadbalancer[LoadBalancer]
+service creates an AWS Elastic Load Balancer (ELB) for your {product-title} cluster, enabling direct TCP access to applications exposed by your LoadBalancer service.
+
+[NOTE]
+====
+LoadBalancer services require an additional purchase. Reach out to your sales team if you're interested in using LoadBalancer services for your {product-title} cluster.
+====
+
+=== Check your LoadBalancer Quota
+
+By purchasing LoadBalancer services, you're provided with a quota of LoadBalancers available for your {product-title} cluster.
+
+----
+$ oc describe clusterresourcequota service-loadbalancers
+Name:       service-loadbalancers
+Labels:     <none>
+Annotations:    <none>
+Resource        Used    Hard
+--------        ----    ----
+services.loadbalancers  0   4
+----
+
+=== Expose TCP Service
+
+You can expose your applications over an external LoadBalancer service, enabling access over the public Internet.
+
+----
+$ oc expose dc httpd-example --type=LoadBalancer --name=lb-service
+service/lb-service exposed
+----
+
+=== Create an Internal-Only TCP Service
+
+You can alternatively expose your applications internally only, enabling access only through AWS VPC Peering or a VPN connection.
+
+----
+$ oc expose dc httpd-example --type=LoadBalancer --name=internal-lb --dry-run -o yaml | awk '1;/metadata:/{ print "  annotations:\n    service.beta.kubernetes.io/aws-load-balancer-internal: 0.0.0.0/0" }' | oc create -f -
+service/internal-lb exposed
+----
+
+=== Use your TCP Service
+
+Once you've created your LoadBalancer service, you can access your service by using the URL provided to you by {product-title}.
+The `LoadBalancer Ingress` value is a URL unique to your service that remains static as long as the service is not deleted. You
+may create a CNAME DNS record for this URL if you prefer to use a custom domain.
```

Comment:
If you prefer to use a custom domain, you can create a CNAME DNS record for this URL.